docs/nodes/credentials/Slack/README.md

```diff
@@ -10,15 +10,30 @@ You can find information about the operations supported by the Slack node on the
 
 Create a [Slack](https://slack.com/) account.
 
+## Using OAuth
+
+1. Access your Slack dashboard.
+2. Click on your username in the top left.
+3. Click the 'Settings & Administration' option, and then 'Manage apps'.
+4. Click on 'Build' in the top right which brings you to another page.
+5. Click on 'Create New App' button.
+6. Enter app name and select your desired workspace.
+7. Scroll down and you will see your authentication information under the *App Credentials* section.
+8. Copy and paste ***Client ID*** and ***Client Secret*** in the Slack OAuth2 API credentials in n8n.
```

Comment:
Could you check and make sure that this is consistent with the rest of the documentation?